### ISSUE TITLE
Vortex can't open external links

### ISSUE THREAD

**Issue by MattSturgeon** [rockerbacon/lutris-skyrimse-installers#7]
When I click a link (e.g. `Open on Nexus Mods` or `Go Premium`) I get the following output and then nothing happens

```
/usr/lib/chromium/chromium: relocation error: /usr/lib/libavformat.so.58: symbol avpriv_tak_parse_streaminfo version LIBAVCODEC_58 not defined in file libavcodec.so.58 with link time reference
```

**Comment by rockerbacon** [comment]
That's a Chromium error. Check if all your software is up to date and if updating doesn't work try setting a non Chromium based browser as your default (Firefox, for example).

**Comment by MattSturgeon** [comment]
Thanks for the pointer.

Using firefox works. It's strange since chromium works fine when using `xdg-open` outside of wine...
<details><summary><strong>Extra info</strong></summary>

```
$ xdg-open http://example.com
Opening in existing browser session.
```

The actual error seems to be related to a ffmpeg library:

```
$ ldd $(which ffmpeg) | grep libavcodec 
	libavcodec.so.58 => /usr/lib/libavcodec.so.58 (0x00007efbfdb2e000)
```

```
$ pacman -Ss libav
extra/ffmpeg 1:4.2.1-2 [installed]
    Complete solution to record, convert and stream audio and video
extra/gst-libav 1.16.1-1 [installed]
    GStreamer open-source multimedia framework FFmpeg plugin
extra/libavc1394 0.5.4-3 [installed]
    A library to control A/V devices using the 1394ta AV/C commands.
community/ffms2 2.23.1-3
    A libav/ffmpeg based source library and Avisynth plugin for easy frame
    accurate access
multilib/lib32-libavc1394 0.5.4-2 [installed]
    A library to control A/V devices using the 1394ta AV/C commands (32-bit)
```

Purely speculation, but I wonder if maybe the library path gets borked somehow at some point?

</details>

---

Changing the wine version in my lutris config from `lutris-4.13-x86_64 (default)` to `System (4.16 (staging))` fixes the issue, so I suspect it is caused by a upstream wine bug fixed between 4.13 and 4.16.

Perhaps this installer should specify a minimum required wine version? I'll leave this open for you to decide if this is something you can/will/should try to fix.

**Comment by rockerbacon** [comment]
The installer does specify a Wine version, current release has it set to lutris-4.12.1-x86_64. I'll update it.
